ansible-ipi-install/roles/node-prep/templates/install-config.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+baseDomain: {{ domain }}
+metadata:
+  name: {{ cluster }}
+networking:
+  machineCIDR: {{ extcidrnet }}
+compute:
+- name: worker
+  replicas: {{ numworkers }}
+controlPlane:
+  name: master
+  replicas: {{ nummasters }}
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: {{ apivip }}
+    ingressVIP: {{ ingressvip }}
+    dnsVIP: {{ ns1vip }}
+    hosts:
+{% for host in groups['masters'] %}
+      - name: {{ hostvars[host]['name'] }}
+        role: {{ hostvars[host]['role'] }}
+        bmc:
+          address: ipmi://{{ hostvars[host]['ipmi_address'] }}
+          username: {{ hostvars[host]['user'] }}
+          password: {{ hostvars[host]['password'] }}
+        bootMACAddress: {{ hostvars[host]['mac'] }}
+        hardwareProfile: default
+{% endfor %}
+{% for host in groups['workers'] %}
+      - name: {{ hostvars[host]['name'] }}
+        role: {{ hostvars[host]['role'] }}
+        bmc:
+          address: ipmi://{{ hostvars[host]['ipmi_address'] }}
+          username: {{ hostvars[host]['user'] }}
+          password: {{ hostvars[host]['password'] }}
+        bootMACAddress: {{ hostvars[host]['mac'] }}
+        hardwareProfile: unknown
```

Comment:
although i ignore whether it would work, a profile different than unknown might be needed for some specific workers(depending on their hardware)

Reply:
@karmab - yes we probably need to discuss. I went by @e-minguez mentioning in a previous thread needing that definition. Aside from all that, I had discussions today with Julia Kreger and she mentioned that from her point of view the installer should only handle installations of the masters and workers should be added later. I plan to discuss with installer team their intentions regarding this.